src/components/Nav.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import "./Nav.scss"
+import {NavLink} from "react-router-dom";
+import {FaHandHoldingHeart} from "react-icons/fa";
+
+
+function Nav() {
+    return (
+        <>
+            <div className="navbar">
+
+                <div className="nav__logo"> "
```

Comment:
Little BEM naming thing: maybe change nav__logo to navbar__logo, as navbar is the name of the Block (B in BEM), and logo is the Element (E in BEM)